src/main/java/com/cupid/jikting/member/entity/Role.java

```diff
@@ -0,0 +1,13 @@
+package com.cupid.jikting.member.entity;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum Role {
+
+    GUEST("ROLE_GUEST"), UNCERTIFIED("ROLE_UNCERTIFIED"), CERTIFIED("ROLE_CERTIFIED");
```

Comment:
p1. 각 필드는 줄 바꿈하는게 좋을 것 같아요!

```suggestion
    GUEST("ROLE_GUEST"),
    UNCERTIFIED("ROLE_UNCERTIFIED"),
    CERTIFIED("ROLE_CERTIFIED");
```